test/integration/policy_zts_cmc_test.go

```diff
@@ -111,13 +111,13 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the zts-cmc policy", Ordered,
 	})
 
 	AfterAll(func() {
-		out, _ := utils.KubectlWithOutput(
-			"get", "deployment", "-n", deploymentNS, deploymentName, "--kubeconfig="+kubeconfigHub,
+		_, err := utils.KubectlWithOutput(
+			"get", "deployment", "-n", deploymentNS, deploymentName, "--kubeconfig="+kubeconfigManaged,
 		)
-		Expect(out).To(ContainSubstring(deploymentName))
+		Expect(err).To(BeNil())
 
-		_, err := utils.KubectlWithOutput(
-			"delete", "deployment", "-n", deploymentNS, deploymentName, "--kubeconfig="+kubeconfigHub,
+		_, err = utils.KubectlWithOutput(
```

Comment:
Should the delete happen after the policy is deleted?

Reply:
Good point!

Reply:
I'm also not sure why the `get` was in there, so I removed it.